src/webpack.js

```diff
@@ -204,12 +204,34 @@ module.exports = async ({ entrypoint, serviceName = 'test-service', ...options }
 
   const outputDir = path.resolve(options.outputPath || process.cwd());
 
+  const removeAsyncAwaitLoader = {
+    loader: 'babel-loader',
+    options: {
+      presets: [ babelEnvConfig ],
+      plugins: [
+        // X-Ray tracing cannot currently track execution across
+        // async/await calls. The issue is tracked upstream at
+        // https://github.com/aws/aws-xray-sdk-node/issues/12 Using
+        // transform-async-to-generator will convert async/await into
+        // generators which can be traced with X-Ray
```

Comment:
We may want to revisit this at some point. I believe that tools like `koa` have deprecated generator support.

Reply:
The transformer does not change the API signature of methods, it uses generators internally and then wraps them in a promise. Not sure why the examples on the plugin webpage look like they do